<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>537</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>412</width>
    <height>537</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>412</width>
    <height>537</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>412</width>
    <height>537</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Figtree</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true">DailyQuests</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listQuests">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>371</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Book Antiqua</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressExpBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bookman Old Style</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineTask">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>109</y>
      <width>301</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Book Antiqua</family>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonAddQuest">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>109</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Book Antiqua</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtLevel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Book Antiqua</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Lv.</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtExpLeft">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Book Antiqua</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Exp until next lvl: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonEasy">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Book Antiqua</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>EASY</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonMedium">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Book Antiqua</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>MEDIUM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonHard">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Book Antiqua</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>HARD</string>
    </property>
   </widget>
   <widget class="QLabel" name="level">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Book Antiqua</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>lvl</string>
    </property>
   </widget>
   <widget class="QLabel" name="expLeft">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Book Antiqua</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>exp</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>412</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
